src/ui/atoms/fileInput/FileInput.tsx

```diff
@@ -8,35 +8,35 @@ import { truthy } from 'utils'
 
 export type FileInputProps = {
   /**
-   * The main title
+   * The main title of the file input.
    */
   readonly label?: string
   /**
-   * Context information for the user
+   * Context information for the user.
    */
   readonly description?: JSX.Element
   /**
-   * Drop one or several files
+   * Drop one or several files.
    */
   readonly multiple?: boolean
   /**
-   * The list of the accepted formats
+   * The list of the accepted formats.
    */
   readonly acceptedFormats?: string[]
   /**
-   * The size of the component
+   * The size of the component.
```

Comment:
I would have added more precision about the size to add more context to the `FilePicker` usage
```suggestion
   * The size of the file input area.
```